rosidl_generator_cpp/resource/msg__traits.hpp.em

```diff
@@ -67,6 +71,32 @@ template<>
 struct has_fixed_size<@(cpp_namespace)@(spec.base_type.type)>
   : std::integral_constant<bool, @(fixed_template_string)>{};
 
+@{
+bounded_template_strings = []
+bounded = True
```

Comment:
Sorry, my example was lacking an important part. I updated the previous comment and added the missing `else` to the `for` loop.